core/storage.go

```diff
@@ -36,6 +44,21 @@ func (n Network) ForkVersion() []byte {
 	}
 }
 
+// GenesisTime returns the genesis time of the network.
+func (n Network) GenesisTime() time.Time {
+	switch n {
+	case TestNetwork:
+		genesisTime, _ := time.Parse(genesisTimeFormat, "2020-11-18 12:00:07 UTC")
```

Comment:
I think it's better to create variables with genesis time on a package level and return that because if the current situation, static data (i.e. time string) gonna be parsed every time.

Something like:
```go
var (
    testNetGenesisTime = time.Date(2020, 11, 18, 12, 0, 7, 0, 0, time.UTC)
)
```

Reply:
> I think it's better to create variables with genesis time on a package level

that's not done

Reply:
got you